PWGHF/vertexingHF/AliAnalysisTaskSEHFSystPID.cxx

```diff
@@ -68,6 +68,17 @@ fCutGeoNcrNclLength(130.),
 fCutGeoNcrNclGeom1Pt(1.5),
 fCutGeoNcrNclFractionNcr(0.85),
 fCutGeoNcrNclFractionNcl(0.7),
+fCutMinCascRadius(1.),
+fCutMinV0Radius(3.),
+fCutMaxV0Radius(85.),
+fCutCosPA(0.995),
+fCutDcaBachToPV(0.1), 
+fCutDcaV0ToPV(0.1),
+fCutDcaV0DaughToPV(0.2),
+fDcaV0Daught(0.1),
+fDcaCascDaught(0.1),
+fCutInvMassLam(0.005),
+fCutNSigmaPID(4.),
```

Comment:
I would suggest to remove this: the idea of the task is to select strange particles without using the PID information, otherwise the PID distributions cannot be studied anymore